relay-server/src/envelope/content_type.rs

```diff
@@ -53,6 +55,7 @@ impl ContentType {
             Self::Protobuf => "application/x-protobuf",
             Self::LogContainer => "application/vnd.sentry.items.log+json",
             Self::SpanV2Container => "application/vnd.sentry.items.span.v2+json",
+            Self::TraceMetricContainer => "application/vnd.sentry.items.metric+json",
```

Comment:
If we use `trace_item` everywhere, I'd also use it here:


```suggestion
            Self::TraceMetricContainer => "application/vnd.sentry.items.trace-metric+json",
```

Using `-` instead of `_` seems to be more common in the registered list of mime types, but I am fine with `_` as well, not sure what we should use here.